docs/source/configuration.rst

```diff
@@ -82,11 +82,39 @@ The ``logging`` section provides directives for logging messages which are usefu
   logging:
       level: ERROR  # the logging level (see https://docs.python.org/3/library/logging.html#logging-levels)
       logfile: /path/to/pygeoapi.log  # the full file path to the logfile
+      logformat: # example for miliseconds:'[%(asctime)s.%(msecs)03d] {%(pathname)s:%(lineno)d} %(levelname)s - %(message)s'
+      dateformat: # example for miliseconds:'%Y-%m-%dT%H:%M:%S'
 
 .. note::
    If ``level`` is defined and ``logfile`` is undefined, logging messages are output to the server's ``stdout``.
 
 
+``logfile rotation``
+^^^^^^^^^^^
+
+The ``logfile rotation`` supports rotation of disk log files. The ``logfile`` file is opened and used as the stream for logging.
+
+.. code-block:: yaml
+
+  logging:
+      logfile: /path/to/pygeoapi.log  # the full file path to the logfile
+      rotation:
+          mode: # [None|time|size]
```

Comment:
Currently if the `mode` is set to `null` in yaml, which would correspond to python `None,` this would lead to an exception because it would be detected as an unknown mode.

I think the code is fine though, and we could require here in the documentation that mode must only be `time|size`. If someone doesn't want log rotation, they can just not have the whole `rotation` block.